src/v2/Apps/ViewingRoom/__tests__/ViewingRoomApp.jest.tsx

```diff
@@ -4,6 +4,7 @@ import ViewingRoomApp from "../ViewingRoomApp"
 import { graphql } from "react-relay"
 import { ViewingRoomApp_OpenTest_QueryRawResponse } from "v2/__generated__/ViewingRoomApp_OpenTest_Query.graphql"
 import { ViewingRoomApp_ClosedTest_QueryRawResponse } from "v2/__generated__/ViewingRoomApp_ClosedTest_Query.graphql"
+import { ViewingRoomApp_UnfoundTest_QueryRawResponse } from "v2/__generated__/ViewingRoomApp_UnfoundTest_Query.graphql"
```

Comment:
`Unfound` is a funny word:) `NotFound` would be more explicit but much more boring. So 👍 to keep it `Unfound`!